packages/db/src/index.ts

```diff
@@ -19,6 +19,11 @@ export const createKysely = (env: Environment) =>
       }),
     }),
     plugins: [new CamelCasePlugin()],
+    // Uncomment this to log all queries, useful for debugging
```

Comment:
Make this available as an env-variable? e.g. `SQL_LOGGING=1`

Reply:
Good suggestion but IMO it's just as easy to just uncomment this in the very few cases that you need it?

Reply:
It is _much_ more practical to use configuration to configure something https://12factor.net/config

Reply:
agreed

Reply:
just use process.env don't make them required

Reply:
Yeah true

Reply:
fixed